docs/_includes/chart.html

```diff
@@ -0,0 +1,602 @@
+                                y="0" patternTransform="rotate(-60 50 50)" id="dataVizPattern1">
+                                <rect x="0" y="0" width="4.5" height="3" fill="var(--color-data-viz-chart-tertiary-background)">
+                                </rect>
+                                <path d="M0 0 L0 3" stroke="var(--color-data-viz-chart-tertiary-stroke)" stroke-width="2"
+                                    fill="none"></path>
+                            </pattern>
+                        </defs>
+                        <path class="donut-graph__segment donut-graph__segment--primary"
+                            d="M 95 11.299999999999997 A 83.7 83.7 0 0 1 112.40220852144647 176.87095418141953 Q 111.25741519711167 177.10594650272733 111.7090100636981 179.3866672389142 L 112.61219979687097 183.94810871128794 Q 113.0637946634574 186.22882944747482 114.33578724605164 185.96772686824391 A 93 93 0 0 0 95 2 Q 93.70151722845951 2.0090652671347016 93.73397929774802 4.333838635456345 L 93.79890343632505 8.98338537209959 Q 93.83136550561356 11.308158740421234 95 11.299999999999997"
+                            role="graphics-symbol" tabindex="0"
+                            aria-labelledby="ebay-chart-c6-chart-legend-label-0"></path>
+                        <path class="donut-graph__segment donut-graph__segment--tertiary"
+                            d="M 100.8386168523833 178.4961110067473 A 83.7 83.7 0 0 1 15.396569586095652 69.13527757081687 Q 15.765456928316212 68.0263613092062 13.564497398547218 67.27709356779525 L 9.162578339009244 65.77855808497338 Q 6.96161880924025 65.02929034356244 6.5517439845507255 66.26141952312986 A 93 93 0 0 0 101.48735205820367 187.77345667416364 Q 102.78203942990534 187.6738359102039 102.5874884441577 185.35699001244882 L 102.19838647266243 180.72329821693862 Q 102.0038354869148 178.40645231918353 100.8386168523833 178.4961110067473"
+                            role="graphics-symbol" tabindex="-1"
+                            aria-labelledby="ebay-chart-c6-chart-legend-label-1"></path>
+                        <path class="donut-graph__segment donut-graph__segment--secondary"
+                            d="M 19.77093832475971 58.308335013754245 A 83.7 83.7 0 0 1 83.35121144964256 12.114562646330555 Q 84.50960835066778 11.959999499978196 84.21820858263077 9.653332819422033 L 83.63540904655676 5.0399994583097225 Q 83.34400927851975 2.73333277775356 82.05690161071396 2.9050696070339512 A 93 93 0 0 0 11.412153694177462 54.23148334861583 Q 10.851084120660175 55.40252588444828 12.954807017643674 56.39246273733707 L 17.162252811610657 58.372336443114655 Q 19.265975708594155 59.36227329600345 19.77093832475971 58.308335013754245"
+                            role="graphics-symbol" tabindex="-1"
+                            aria-labelledby="ebay-chart-c6-chart-legend-label-2"></path>
+                    </svg></div>
+                <div class="chart-legend__container chart-legend__container-vertical">
```

Comment:
Do we need `chart-legend__container-vertical` if vertical is the default? Also, syntax should be a double-dash, not single dash.

Reply:
removing it